prisma/migrations/20230307211548_add_ip_address_to_redemption/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `ip_address` to the `redemptions` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "redemptions" ADD COLUMN     "ip_address" VARCHAR NOT NULL;
```

Comment:
Does this need to be backfilled?

```
ironfish-api-production::DATABASE=> SELECT COUNT(*) FROM redemptions;
 count
-------
    20
(1 row)
```

Reply:
Lets make it nullable, I think this is a better approach. Honestly I am a bit surprised the prisma migration system didn't pick up on this?

Reply:
Prisma adds a warning to the top of the file. I don't think the ORM can raise anything beyond that unless you point it at a db.

Reply:
I would have expected it to error out like Django with something like "Creating a nonnullable column on a model that already exists, either provide default value or make column nullable" 🤷